.gitignore

```diff
@@ -5,6 +5,7 @@
 *.suo
 *.user
 *.sln.docstates
+.idea
```

Comment:
I do believe the file (.gitignore) is added by accident.
If not, please provide the argument.